modules/analytics/con-telemetry-data-collection-and-analysis.adoc

```diff
@@ -0,0 +1,34 @@
+[id="con-telemetry-data-collection-and-analysis_{context}"]
+= Telemetry data collection and analysis
+
+The telemetry data collection feature helps in collecting and analyzing the telemetry data to improve your experience with {product}. This feature is enabled by default.
+
+{company-name} collects and analyzes the following data:
+
+Web Analytics::
+
+* Web Analytics use the Segment tool.
+
+* It is the tracking of user behavior and interactions with {product}.
+
+Specifically, it tracks the following:
+
```

Comment:
Attempt to fix formatting. See https://docs.asciidoctor.org/asciidoc/latest/lists/description/

```suggestion
Web Analytics::
Web Analytics use the Segment tool.
It is the tracking of user behavior and interactions with {product}.
Specifically, it tracks the following:
```